farmerbot/README.md

```diff
@@ -0,0 +1,207 @@
+# Farmerbot
+
+<a href='https://github.com/jpoles1/gopherbadger' target='_blank'>![gopherbadger-tag-do-not-edit](https://img.shields.io/badge/Go%20Coverage-73%25-brightgreen.svg?longCache=true&style=flat)</a>
+
+Farmerbot is a service that a farmer can run allowing him to automatically manage power of the nodes of his farm.
+
+## How to use
+
+- Create a new yml/yaml file `config.yml` and add your configurations:
+
+```yml
+farm_id: "<your farm ID, required>"
+included_nodes:
+  - "<your node ID to be included, required at least 2>"
```

Comment:
its IDs in plural

Reply:
It's `ID` because it is an item of the list